packages/electrode-react-webapp/lib/index.js

```diff
@@ -106,11 +106,22 @@ function makeRouteHandler(options, userContent) {
   const criticalCSS = getCriticalCSS(options.criticalCSS);
 
   /* Create a route handler */
-  /* eslint max-statements: [2, 20] */
+  /* eslint max-statements: [2, 25] */
   return (request, reply) => {
     const mode = request.query.__mode || "";
+    const disableSsrFlag = request.server.app && request.server.app.disableSSR;
     const renderJs = RENDER_JS && mode !== "nojs";
-    const renderSs = RENDER_SS && mode !== "noss";
+    let renderSs = RENDER_SS;
+    if (renderSs) {
+      if (mode === "noss") {
+        renderSs = false;
+      } else if ((mode === "datass" || disableSsrFlag) && request.app) {
+        request.app.disableSSR = true;
+      } else if (request.app && request.app.disableSSR) {
+        request.app.disableSSR = false;
```

Comment:
this logic here doesn't make sense. could you explain the reason at flipping `disableSSR` from `true` to `false`?